suffuse/process.go

```diff
@@ -1,7 +1,7 @@
 package suffuse
 
 import (
-  "github.com/codeskyblue/go-sh"
```

Comment:
I do agree that this dependency is buying us zero. That looks to be a common theme with go dependencies, that they're often no improvement on nothing.